tex/gregoriotex-main.tex

```diff
@@ -560,18 +620,25 @@
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
 % format for text above the lines
-\def\greabovelinestextstyle#1{%
-#1\relax %
+\def\greabovelinestextstyle#1{% DEPRECATED
+  \relax %
 }%
 
 % set space above the text lines - almost the same as for the translation
 \def\greaddspaceabove{%
-  \gre@style@abovelinestext%
-  \greabovelinestextstyle{%
-    \global\gre@dimen@currentabovelinestextheight=\gre@dimen@abovelinestextheight %
-    \gregeneratelines %
-  }%
-  \endgre@style@abovelinestext%
+  \protected@edef\gre@@arg{\greabovelinestextstyle{a}}%  DEPRECATED
+  \expandafter\ifx\gre@@arg\relax% DEPRECATED
```

Comment:
You pulled a similar trick in implementing the point-and-click feature.  When I came across it while doing the documentation on the texrenaming branch it jostled my memory about this issue and I did some experimentation to see if I could adapt it to this application.  This is the result.